Sources/FINNFilterImplementation/FilterMarkets/FilterMarketBap.swift

```diff
@@ -31,6 +31,15 @@ extension FilterMarketBap: FilterConfiguration {
         return .location
     }
 
+    func contextFilterKeys(for key: FilterKey) -> [FilterKey] {
+        switch key {
+        case .category:
```

Comment:
Why are you grouping the context filters to another filter? This isn't really how they work...